packages/cashier/src/components/usdt-side-note/usdt-side-note.tsx

```diff
@@ -1,77 +0,0 @@
-import React from 'react';
-import { Modal, Text } from '@deriv/components';
-import { Localize } from '@deriv/translations';
-import './usdt-side-note.scss';
-
-type TUSDTSideNoteProps = {
-    type: 'eusdt' | 'usdt';
-};
-
-const USDTSideNote = ({ type }: TUSDTSideNoteProps) => {
```

Comment:
The last use was replaced with `DepositCryptoSideNoteUSDT` in this PR, Hence removing the `deprecated` component.